install/prereqs/centos.rst

```diff
@@ -4,47 +4,59 @@
 RedHat / CentOS system prerequisites
 ####################################
 
+..   yum install bison curl blas bzip2-devel bzip2 flex fontconfig \
+       freetype-devel gcc-c++ gcc-gfortran git libuuid-devel \
+       libXext libXrender libXt-devel make openssl-devel patch perl \
+       readline-devel tar zlib-devel ncurses-devel cmake glib2-devel \
+       java-1.8.0-openjdk gettext perl-ExtUtils-MakeMaker \
+       centos-release-scl devtoolset-6-gcc devtoolset-6-gcc-c++ \
+       devtoolset-6-gcc-gfortran mesa-libGL-devel
```

Comment:
Do we need the commented block above? I'm not sure what it's for. Remove it unless it's necessary.

Reply:
I remove the commented block. It is not needed indeed.